extensions/cornerstone/src/commandsModule.js

```diff
@@ -186,7 +186,7 @@ const commandsModule = ({ servicesManager, commandsManager }) => {
         }
 
         const viewportInfo = getEnabledElement(i);
-        const hasCornerstoneContext =
+        const hasCornerstoneContext = viewportInfo.context &&
```

Comment:
This was just an annoying error message that occasionally popped up due to load timing.